airflow/providers/openlineage/plugins/adapter.py

```diff
@@ -329,19 +330,26 @@ def dag_started(
         self,
         dag_run: DagRun,
         msg: str,
-        nominal_start_time: str,
-        nominal_end_time: str,
```

Comment:
Instead of building it in listener, we moved it to the inside of the adapter's method. I'm okay with that, but not sure if the adapter methods are part of "public" api and it's `breaking` change @mobuchowski ?

Reply:
Reverted.

But IMHO to be a public api this method should be at least properly documented. And maybe even redesigned, as it currently accepts DagRun, unlike other methods which do not accept Airflow models, but instead set of separated fields to just build OpenLineage events from.

Reply:
Adapter methods are not public API.

Reply:
@dolfinus it's more likely we'll redesign the whole mechanism in Airflow 3 due to AIP-72 - and possibly stop running on worker altogether.